vxfreeswitch/voice.py

```diff
@@ -50,6 +51,15 @@ def __init__(self, vumi_transport):
         self.input_type = None
         self.uniquecallid = None
 
+    # NOTE: These were lifted straight from eventsocket's github code
+    #       Once https://github.com/fiorix/eventsocket/issues/11 is closed
+    #       This code (and the vumi.log import) can be undone
+    def unknownContentType(self, content_type, ctx):
+        log.debug("[eventsocket] unknown Content-Type: %s" % content_type)
+
+    def unboundEvent(self, ctx, evname):
+        log.debug("[eventsocket] unbound Event: %s" % evname)
```

Comment:
When it's not run via Junebug it will log to the normal stderr / stdout. The only thing that this does is set the `system` of the logger to the `worker_name`. Junebug can then look at the `system` of each incoming log, and route it to the correct place to have separate logs for each channel. By default these things will go to stdout / stderr.